app/controllers/autotitle_controller.rb

```diff
@@ -0,0 +1,4 @@
+class AutotitleController < ApplicationController
+  def get_url
```

Comment:
Keep it RESTful. i.e. unless it's a strange situation, stick with convention instead of creating a non-standard controller action (i.e. anything other than `new`, `create`, `edit`, `update`, etc).